still search in /etc/magic
still use ~/.magic by default

--- file-5.04/src/Makefile.am	2010-07-21 18:56:10.000000000 +0300
+++ file-5.04/src/Makefile.am	2010-12-23 13:27:13.810960488 +0200
@@ -4,7 +4,7 @@
 
 bin_PROGRAMS = file
 
-AM_CPPFLAGS = -DMAGIC='"$(MAGIC)"'
+AM_CPPFLAGS = -DMAGIC='"/etc/magic:$(MAGIC)"'
 AM_CFLAGS = @WARNINGS@
 
 libmagic_la_SOURCES = magic.c apprentice.c softmagic.c ascmagic.c \
--- file-5.06/src/magic.c~	2011-04-07 22:20:30.000000000 +0300
+++ file-5.06/src/magic.c	2011-04-18 11:56:38.611292743 +0300
@@ -107,7 +107,7 @@
 private const char *
 get_default_magic(void)
 {
-	static const char hmagic[] = "/.magic/magic.mgc";
+	static const char hmagic[] = "/.magic";
 	static char *default_magic;
 	char *home, *hmagicpath;
 
